### ISSUE TITLE
Accuracy results on cifar100

### ISSUE THREAD

**Issue by shuikehuo** [google/bi-tempered-loss#4]
this paper reports 75.30 accuracy on the clean test set. But I obatin 78.16 accuracy on the same test set, I use resnet50 with SGD + momentum optimizer trained for 350 epoch.

**Comment by rohan-anil** [comment]
Hi Shuikehuo,

We used Resnet-56 without batch norm from [1] which explains the accuracy difference (and a weaker baseline). And it was trained with SGD optimizer for 50k steps with batch size 128.

The experiment shows the effect of noisy labels on the test accuracy when trained with logistic loss, and with bi-tempered logistic loss. We expect that the results in terms of accuracy delta to remain similar even when trained with the Resnet-50 (with batch norm) model or models of similar capacity.  We will make available the code for Resnet-56 model without batch norm from [1] soon to reproduce the results.

Thanks,

[1] Identity Matters in Deep Learning, Moritz Hardt, Tengyu Ma, https://arxiv.org/pdf/1611.04231.pdf

**Comment by Charles-Xie** [comment]
> Hi Shuikehuo,
> 
> We used Resnet-56 without batch norm from [1] which explains the accuracy difference (and a weaker baseline). And it was trained with SGD optimizer for 50k steps with batch size 128.
> 
> The experiment shows the effect of noisy labels on the test accuracy when trained with logistic loss, and with bi-tempered logistic loss. We expect that the results in terms of accuracy delta to remain similar even when trained with the Resnet-50 (with batch norm) model or models of similar capacity. We will make available the code for Resnet-56 model without batch norm from [1] soon to reproduce the results.
> 
> Thanks,
> 
> [1] Identity Matters in Deep Learning, Moritz Hardt, Tengyu Ma, https://arxiv.org/pdf/1611.04231.pdf

@rohan-anil Is there any reason to use resnet56 without batch normalization? This network seems not to be used a lot in experiments. 

When I use resnet110 with BN (as introduced in ResNet v1 paper), the accuracy delta (improvement) does not seems to be very obvious, for clean or noisy labels. 

![result](https://user-images.githubusercontent.com/24546725/71675512-de001080-2db8-11ea-880d-a6d9a6a33511.jpg)

**Comment by eamid** [comment]
Hi Chi,

Thank you for your interest in our method.

We used the Resnet-56 model because we had the baseline easily available (Moritz was at google, and we used his codebase). I noticed that the bi-tempered loss still gives some improvements in your case. You might achieve even more improvement by tuning t1 and t2 (I would suggest trying a larger t2 value). 

Ehsan

**Comment by Charles-Xie** [comment]
@eamid 
Thanks a lot!
